index.html

```diff
@@ -0,0 +1,366 @@
+                <span
+                  >Not properly updating your systems makes you an easy to reach
+                  target and hackers will often go after the lowest hanging
+                  fruit. When it comes to maintaining a strong cybersecurity
+                  network the best thing a user can do is to prevent problems
+                  before they occur, and the easiest way to do that is by making
+                  sure that all systems are updated and updated regularly.
+                </span>
+              </div>
+              <div class="bottomInfo">
+                <img src="image/Letter_i.png" alt="#" />
+                <span>Based on the 2018 Verizon Data Breach Report</span>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="line"></div>
+        <div class="accordContainer">
+          <div class="headAccordion">
+            <span class="number">4</span>
+            <span class="accordText">Secure Credentials</span>
+            <button class="accordion"></button>
+          </div>
+          <div class="panel">
+            <img src="image/groop4.png" alt="#" class="mainIcon" />
+            <div class="footerAccordionContainer">
+              <div class="footerAccordion">
+                <span
+                  >There’s nothing hackers love more than a dismissed update
+                  reminder. We keep our team’s computers in shape and up to date
+                  so that they aren’t subject to attacks that could otherwise be
+                  avoided.</span
+                >
+                <span
+                  >Not properly updating your systems makes you an easy to reach
+                  target and hackers will often go after the lowest hanging
+                  fruit. When it comes to maintaining a strong cybersecurity
+                  network the best thing a user can do is to prevent problems
+                  before they occur, and the easiest way to do that is by making
+                  sure that all systems are updated and updated regularly.
+                </span>
+              </div>
+              <div class="bottomIcon">
+                <img src="image/Letter_i.png" alt="#" />
+                <span>Based on the 2018 Verizon Data Breach Report</span>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="line"></div>
+
+        <div class="glossaryContaner">
+          <div class="line"></div>
+          <img src="image/skout-glossary.svg" id="headGlossary" alt="#" />
+        </div>
+      </div>
+
+      <section>
+        <div class="sliderContainer">
+          <div class="sliderHead"></div>
+          <div class="sliderButtonContainer">
+            <button class="sliderButton" value="right"></button>
+            <button class="sliderButton rotateArrow" value="left"></button>
+          </div>
+        </div>
+        <div class="sliderTextContainer">
+          <div>
```

Comment:
extra wrapper because `p` is a block element

Reply:
done

Reply:
not fixed